<?xml version="1.0"?>
<launch>
<arg name="robot" default="$(optenv ROBOT !!NO_ROBOT_SET!!)"/>

<node pkg="nodelet" type="nodelet" name="camera_throttle"  args="load cob_camera_sensors/Cam3DThrottle camera_nodelet_manager" output="screen">
	<remap from="rgb_cloud_in" to="/camera/depth_registered/points"/>
	<remap from="rgb_image_in" to="/camera/rgb/image_raw"/>
	<remap from="rgb_caminfo_in" to="/camera/rgb/camera_info"/>
	<remap from="depth_image_in" to="/camera/depth/image_raw"/>
	<remap from="cloud_in" to="/camera/depth/points"/>
	<remap from="rgb_cloud_out" to="/camera/depth_registered/points_throttled"/>
	<remap from="rgb_image_out" to="/camera/rgb/image_raw_throttled"/>
	<remap from="rgb_caminfo_out" to="/camera/rgb/camera_info_throttled"/>
	<remap from="depth_image_out" to="/camera/depth/image_raw_throttled"/>
	<remap from="cloud_out" to="/camera/depth/points_throttled"/>
	<param name="max_rate" value="5.0"/>
</node>
</launch>

